src/components/IconTooltip.tsx

```diff
@@ -0,0 +1,56 @@
+import { Theme, Tooltip, TooltipProps } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import Icon from './Icon/Icon';
+import { IconName } from './Icon/icons';
+import { useDeviceInfo } from '../utils';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  arrow: {
+    color: '#3A4445',
+    marginLeft: '3px',
+    marginTop: '-1px',
+  },
+  icon: {
+    fill: '#3A4445',
+    verticalAlign: 'text-top',
+  },
+  tooltip: {
+    backgroundColor: '#3A4445',
+    borderRadius: '8px',
+    padding: '8px',
+    '& a': {
+      color: '#fff',
+    },
+  },
+}));
+
+type IconTooltipProps = {
+  iconName?: IconName;
+  placement?: TooltipProps['placement'];
+  title: TooltipProps['title'];
```

Comment:
I've already wrote this on terraware-web, but could this title be a string?

Reply:
@constanzauanini it should match the type of the `title` prop of the `Tooltip` component from MUI: `NonNullable<React.ReactNode>`

`TooltipProps['title']` === `NonNullable<React.ReactNode>`

Reply:
Yes, i saw that later! Thanks